.github/workflows/labeler.yml

```diff
@@ -6,6 +6,6 @@ jobs:
   triage:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/labeler@v3
+    - uses: actions/labeler@v4
       with:
         repo-token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
Would it still work with the following removal?
```suggestion
```

Reply:
Keeping it won't cause any issue